server/tilesource/tiff.py

```diff
@@ -38,23 +38,28 @@ class TiffGirderTileSource(GirderTileSource):
     @staticmethod
     def cacheKeyFunc(args, kwargs):
         item = args[0]
-        return item.get('largeImage')
+        return item.get('largeImage') if item else None
```

Comment:
I should have removed this.  It was added when I was testing if a file had a large image before adding 'largeImage' to the item (since refactored).